customizeMuonHLTForRun3.py

```diff
@@ -2,6 +2,579 @@
+        dr_par1 = cms.vdouble( 3.40282346639E38, 0.4, 0.4 ),
+        dr_exp = cms.vint32( 4, 4, 4 ),
+        d0err_par = cms.vdouble( 0.001, 0.001, 0.001 )
+      ),
+      maxLostLayers = cms.vint32( 1, 1, 1 ),
+      min3DLayers = cms.vint32( 0, 0, 0 ),
+      dz_par = cms.PSet(
+        dz_par1 = cms.vdouble( 3.40282346639E38, 0.4, 0.4 ),
+        dz_par2 = cms.vdouble( 3.40282346639E38, 0.35, 0.35 ),
+        dz_exp = cms.vint32( 4, 4, 4 )
+      ),
+      minNVtxTrk = cms.int32( 3 ),
+      maxDz = cms.vdouble( 0.5, 0.2, 3.40282346639E38 ),
+      minNdof = cms.vdouble( 1.0E-5, 1.0E-5, 1.0E-5 ),
+      maxChi2 = cms.vdouble( 9999.0, 25.0, 16.0 ),
+      maxChi2n = cms.vdouble( 1.2, 1.0, 0.7 ),
+      maxDr = cms.vdouble( 0.5, 0.03, 3.40282346639E38 ),
+      minLayers = cms.vint32( 3, 3, 3 )
+    )
+    )
+
+    process.hltIter3IterL3MuonTrackSelectionHighPurity = cms.EDProducer( "TrackCollectionFilterCloner",
+    originalSource = cms.InputTag( "hltIter3IterL3MuonCtfWithMaterialTracks" ),
+    originalMVAVals = cms.InputTag( 'hltIter3IterL3MuonTrackCutClassifier','MVAValues' ),
+    originalQualVals = cms.InputTag( 'hltIter3IterL3MuonTrackCutClassifier','QualityMasks' ),
+    minQuality = cms.string( "highPurity" ),
+    copyExtras = cms.untracked.bool( True ),
+    copyTrajectories = cms.untracked.bool( False )
+    )
+
+    ## New track merger
+    process.hltIter03IterL3MuonMerged  = cms.EDProducer( "TrackListMerger",
+        ShareFrac = cms.double( 0.19 ),
+        FoundHitBonus = cms.double( 5.0 ),
+        LostHitPenalty = cms.double( 20.0 ),
+        MinPT = cms.double( 0.05 ),
+        Epsilon = cms.double( -0.001 ),
+        MaxNormalizedChisq = cms.double( 1000.0 ),
+        MinFound = cms.int32( 3 ),
+        TrackProducers = cms.VInputTag( 'hltIter0IterL3MuonTrackSelectionHighPurity','hltIter3IterL3MuonTrackSelectionHighPurity' ),
+        hasSelector = cms.vint32( 0, 0 ),
+        indivShareFrac = cms.vdouble( 1.0, 1.0 ),
+        selectedTrackQuals = cms.VInputTag( 'hltIter0IterL3MuonTrackSelectionHighPurity','hltIter3IterL3MuonTrackSelectionHighPurity' ),
+        setsToMerge = cms.VPSet(
+          cms.PSet(  pQual = cms.bool( False ),
+            tLists = cms.vint32( 0, 1 )
+          )
+        ),
+        trackAlgoPriorityOrder = cms.string( "hltESPTrackAlgoPriorityOrder" ),
+        allowFirstHitShare = cms.bool( True ),
+        newQuality = cms.string( "confirmed" ),
+        copyExtras = cms.untracked.bool( True ),
+        writeOnlyTrkQuals = cms.bool( False ),
+        copyMVA = cms.bool( False )
+    )
+
+    process.hltIterL3MuonMerged.TrackProducers = cms.VInputTag( 'hltIterL3OIMuonTrackSelectionHighPurity','hltIter03IterL3MuonMerged' )
+    process.hltIterL3MuonMerged.selectedTrackQuals = cms.VInputTag( 'hltIterL3OIMuonTrackSelectionHighPurity','hltIter03IterL3MuonMerged' )
+    process.hltL3MuonsIterL3IO.L3TrajBuilderParameters.tkTrajLabel = cms.InputTag( "hltIter03IterL3MuonMerged" )
```

Comment:
Is this part really needed?
You can search hltIter0IterL3MuonTrackSelectionHighPurity in HLT menu. And substitute everything into hltIter03IterL3MuonMerged